modules/k8s-operator-crd-support/main.tf

```
@@ -115,7 +119,8 @@ module "wait_for_gatekeeper" {
   project_id               = var.project_id
   create_cmd_triggers      = { script_sha1 = sha1(file("${path.module}/scripts/wait_for_gatekeeper.sh")) }
   service_account_key_file = var.service_account_key_file
+  use_existing_context     = var.use_existing_context
 
-  kubectl_create_command  = "${path.module}/scripts/wait_for_gatekeeper.sh ${var.project_id} ${var.cluster_name} ${var.location}"
+  kubectl_create_command  = format("%s %s", "${path.module}/scripts/wait_for_gatekeeper.sh ${var.project_id} ${var.cluster_name} ${var.location}", local.append_arg_use_existing_context_for_gatekeeper)
```

nit: Rather than using both specification string and interpolation, can we just use interpolation?
```suggestion
  kubectl_create_command  = "${path.module}/scripts/wait_for_gatekeeper.sh ${var.project_id} ${var.cluster_name} ${var.location} ${local.append_arg_use_existing_context_for_gatekeeper}"
```

---

Sounds good. I will commit the suggestion.